mflowgen/pad_frame/custom-power/outputs/globalnetconnect.tcl

```diff
@@ -0,0 +1,11 @@
+# Same as everyone else
+connect_global_net VDD -type pgpin -pin VDD -inst *
+connect_global_net VDD -type tiehi
+connect_global_net VSS -type pgpin -pin VSS -inst *
+connect_global_net VSS -type tielo
+connect_global_net VDD -type pgpin -pin VPP -inst *
+connect_global_net VSS -type pgpin -pin VBB -inst *
+
+# Added for butterphy maybe
+connect_global_net VDDPST -type pgpin -pin VDDPST -inst *
```

Comment:
This is not for the butterphy. VDDPST and VSS PST are I/O power nets.

Reply:
I updated the comment.